index.html

```diff
@@ -262,36 +265,23 @@ <h6>CANOPY</h6>
               </div>
             </div>
           </div>
-        </div>
-        <section class="about-myself">
-          <div class="top">
-            <div class="primary-text-5">
-              <h1>About <br>
-              Myself</h1>
-            </div>
-              <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <div class="social-media">
-              <div class="supporting-text">
-                <span>LET'S CONNECT!</span>
-              </div>
-            </div>
-            <div class="social-media-icons">
-              <ul class="social-media-icons-list">
-                <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
-                <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
-                <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
-                <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
-                <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
-              </ul>
-            </div>
-            <div class="normal-button-secondary">
-              <div class="get-my-resume">
-                <button>Get my resume</button>
-              </div>
-            </div>
+        </section>
+        <section class="about-myself" id="about">
+          <div class="about-block">
+            <h1 class="about">About <br> myself<br></h1>
+            <p class="about-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <h4 class="let-s-connect-2">LET'S CONNECT!</h4>
```

Comment:
- Please use the proper background color for the body as in the Figma design  which should be ` #E5E5E5;`
- Please make the background of the about section takes full width of the webpage
##### Figma design 
![Screenshot from 2022-06-03 18-34-39](https://user-images.githubusercontent.com/58771507/171916882-da74526b-81d6-44fc-84ea-d77cb50002d3.png)





<br><br>

##### Your design 
![Screenshot from 2022-06-03 18-31-36](https://user-images.githubusercontent.com/58771507/171916943-8d208313-07e6-44fe-a11d-05f4f4607019.png)